crates/forge-runner/src/printing.rs

```diff
@@ -1,60 +1,130 @@
+// TODO(#3022): Rename `printing.rs` to `messages.rs``
```

Comment:
Why not just do it here?

Reply:
If you rename the file (not copy it's contents to another, actually rename) then git will track it and github as well.